domains/auth/Auth.API/API/Repository/CompanyStorage.cs

```diff
@@ -0,0 +1,8 @@
+using API.Models;
+
+namespace API.Repository;
+
+public class CompanyStorage : ICompanyStorage
+{
+    public Task<Company?> CompanyByTin(string tin) => Task.FromResult<Company?>(null);  //FIXME implement this
```

Comment:
Is the comment needed?
I think if we have //TODO/FIXME there should be a link to a story fixing it